apps/project-renderer/src/components/app/index.js

```diff
@@ -20,17 +20,13 @@ import { useStylesheet } from '@crowdsignal/hooks';
 import { setHostOption } from '@crowdsignal/http';
 import { fetchProjectForm } from '@crowdsignal/rest-api';
 
-/**
- * Style dependencies
- */
-import './style.scss';
-
 const App = ( {
 	projectCode,
 	page = 0,
 	preview,
 	respondentId = '',
 	startTime = 0,
+	theme = 'leven',
```

Comment:
In this case, it might not be a bad idea to leave it as a prop here - on the contrary to the editor - as it would allow us to load things sooner.  
That said, we could actually load the themes synchronously on production and only use this code for the development environment. But that's outside of the scope of this PR.